.github/workflows/ci.yaml

```diff
@@ -71,6 +71,8 @@ jobs:
         env:
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
+      - name: Run storybook tests
+        run: cd ./ui && npm run storybook:serve-and-test
```

Comment:
This runs storybook tests using the storybook test runner. https://storybook.js.org/docs/react/writing-tests/test-runner